src/m_menu.c

```diff
@@ -903,7 +903,7 @@ void M_LoadSelect(int choice)
 
   name = G_SaveGameName(choice);
 
-  saveg_compat = saveg_woof510;
+  saveg_compat = saveg_woof1200;
```

Comment:
@rfomin The savegame versioning code is frightening me. Could you please have a second look?